Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -879,12 +879,12 @@
     (require-all
         (require-not (extension "com.apple.webkit.extension.iokit"))
         (iokit-connection "IOGPU")))
-#endif
 
 (deny iokit-open-user-client
```

Comment:
Should this be allow instead of deny after this change? (according to the rules above)

Reply:
That is a good point. In this case, I think we do want to keep the deny, since we are only looking to remove this rule for the case when ENABLE(WEBCONTENT_GPU_SANDBOX_EXTENSIONS_BLOCKING) == 1. There should be no behavior change in either case, since this is denied by default.

Thanks for reviewing!

Reply:
So we